scripts/mgear/core/anim_utils.py

```diff
@@ -331,8 +331,8 @@ def get_ik_fk_controls_by_role(uiHost, attr_ctl_cnx):
                     ik_controls["pole_vector"] = c.stripNamespace()
                 elif role == "ik":
                     ik_controls["ik_control"] = c.stripNamespace()
-                elif role == "ik":
-                    ik_controls["ikRot"] = c.stripNamespace()
+                elif role == "ikRot":
```

Comment:
Not sure why this has changed?  Can someone check what this is meant to be?  "ik" or "ikRot"

Reply:
@thirstydevil ikRot is the sphere control for the ik hand rotation
![image](https://user-images.githubusercontent.com/1050212/124678595-cea5da00-defd-11eb-953a-2789ec2489b7.png)